packages/core-typings/src/IUser.ts

```diff
@@ -185,6 +217,7 @@ export interface IUser extends IRocketChatRecord {
 	_pendingAvatarUrl?: string;
 	requirePasswordChange?: boolean;
 	requirePasswordChangeReason?: string;
+	isOAuthUser?: boolean;
```

Comment:
Aren't we using this only in the ednpoint return? I believe this doesn't need to be in core-typings

Reply:
This is a client only field, on client side we will be using `user?.isOAuthUser` to check this, so it is required here.

I'll put a comment here mentioning this is a client only field.

We don't need this on rest typings since the API returns IUser.